src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/UpdateDao.java

```diff
@@ -43,7 +43,7 @@ public class UpdateDao {
     private List<ManagingOfficerCorporateDao> reviewManagingOfficersCorporate;
 
     @Field("review_trusts")
-    private List<TrustDataDao> reviewTrusts;
```

Comment:
Why are we not storing the Trusts to review? They should be loaded once and then stored for review.

Reply:
Ok. I see this is both the Trust object and the review status now.